### ISSUE TITLE
Native Scrolling Container

### ISSUE THREAD

**Issue by saeder** [codenameone/CodenameOne#2121]
The scrolling behaviour of Codename One components is not anywhere near as good as the native "rubberband" scrolling behaviour in IOS.

I wish there was a native scrolling container.

**Comment by saeder** [comment]
The Codename One Scrolling behaviour is unbearable. See also https://github.com/codenameone/CodenameOne/issues/1998 https://github.com/codenameone/CodenameOne/issues/1736 https://github.com/codenameone/CodenameOne/issues/1735 https://github.com/codenameone/CodenameOne/issues/1365

**Comment by codenameone** [comment]
I suggest reading the documentation and architecture code before filing "issues". This is not an issue https://www.codenameone.com/blog/issue-submission-guideline.html

**Comment by saeder** [comment]
The various disadvantages of the Codename One tensile scrolling affect the user experience in a way that is certainly remarkable. You should see it as an issue.

**Comment by codenameone** [comment]
I closed this because this isn't an issue. It's a rant with a suggestion that shows a deep lack of understanding regarding the underlying architecture of Codename One. 

This is an issue tracker, not a discussion board. I would suggest looking at the basic docs and source code to understand how Codename One works and why I closed this issue. If I thought tensile behavior wasn't worth improving I would have closed your other issues.
